frontend/src/components/CopyableText/CopyableText.tsx

```diff
@@ -11,14 +11,14 @@ export function CopyableText({
   ...props
 }: CopyableTextProps & React.HTMLProps<HTMLDivElement>) {
   return (
-    <div {...props}>
+    <>
```

Comment:
Why remove this? This allows you to style the component etc

Reply:
Because having the two components nested under a div was making them wrap poorly, this way all the elements are on the same level and can be handled consistently by the flexbox.